fluent/protocol/message.go

```diff
@@ -135,6 +153,22 @@ type MessageExt struct {
 	Options   *MessageOptions
 }
 
+// NewMessage creates a Message from the supplied
+// tag and record. It will set Timestamp to
+// time.Now().UTC().Unix().
```

Comment:
Should be: `time.Now().UTC()` 
 ... or `EventTimeNow()`

Reply:
This is correct for this type. MessageExt takes EventTime. The difference is part of the spec